### ISSUE TITLE
Table Storage: Query tables always return not found ERROR

### ISSUE THREAD

**Issue by wankes2000** [arafato/azurite#169]
Hi 

When I try to retrieve one previously created table using Azurite emulator by calling:

```
GET http://127.0.0.1:10002/devstoreaccount1/Tables
```
I always receive the same ERROR response:

```
<?xml version="1.0" encoding="utf-8"?>
<Error>
    <Code>TableNotFound</Code>
    <Message>The table specified does not exist.</Message>
</Error>
```

Is Table Storage query tables feature fully implemented. Thank you in advance

**Comment by arafato** [comment]
Hi @wankes2000 ! Thanks for reporting! Just fixed this bug and released as `v2.6.1`.
The table storage support is almost fully implemented except batch operations and `$select` query syntax. Please be aware, however, that table storage support is currently not tested very well. Will start thorough testing when batch feature is implemented.

**Comment by wankes2000** [comment]
Thank you very much for your quick response
